public/js/util/create-store.js

```diff
@@ -46,10 +62,10 @@ module.exports = (opts = {}) => {
     middleware.push(log);
   }
 
-  // Optionally add the Redux DevTools Extension, if it exists.
-  const devtools = typeof window === "object" && window.devToolsExtension ?
-    window.devToolsExtension() :
-    f => f;
+  // // Optionally add the Redux DevTools Extension, if it exists.
+  // const devtools = typeof window === "object" && window.devToolsExtension ?
+  //   window.devToolsExtension() :
+  //   f => f;
```

Comment:
I don't think we should uncomment this. You should probably check for this first, and if that doesn't exist, use `DevTools`. This lets people with the hook into the app with the redux browser addon.